### PR TITLE
fixes the order and text as per the issue, also localizes for spanish

### LINKED ISSUE #594: adjust help text on "Add New Batch Upload Item" page
On the Files tab of this page, the help text currently reads:

> To create a single work for all the files, go to `<link>`New Work`</link>`
> Each file will be uploaded to a separate new work resulting in one work per uploaded file.

It would be clearer to a user to reverse the order of these statements and modify the text to read as follows:

> Each file will be uploaded to create a separate new work, resulting in one work per file.
> To create a single work for multiple files, use the `<link>`Add New Work`</link>` form.

### PR DESCRIPTION
**Pull request by carrickr**
Fixes #594 

Present tense short summary (50 characters or less)

Makes the requested text changes and implements localization for en and es.

@projecthydra-labs/hyrax-code-reviewers

### CODE REVIEW

File: app/views/hyrax/batch_uploads/_form.html.erb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <%= simple_form_for [hyrax, @form], html: { multipart: true } do |f| %>
   <% content_for :files_tab do %>
-    <p class="switch-upload-type">To create a single work for all the files, go to
-      <%= link_to "New Work", [main_app, :new, Hyrax.primary_work_type.model_name.singular_route_key] %></p>
     <p class="instructions"><%= t("hyrax.batch_uploads.files.instructions") %></p>
+    <p class="switch-upload-type"><%= t("hyrax.batch_uploads.files.upload_type_instructions") %>
+      <%= link_to t("hyrax.batch_uploads.files.label"), [main_app, :new, Hyrax.primary_work_type.model_name.singular_route_key] %>
```
Comment (jcoyne) [inline app/views/hyrax/batch_uploads/_form.html.erb]: Can we call this `hyrax.batch_uploads.files.button_label`?

File: config/locales/hyrax.es.yml [hunk 2]
```diff
@@ -271,6 +271,8 @@ es:
     batch_uploads:
       files:
         instructions: Cada archivo será cargado en un nuevo trabajo separado, resultando en un trabajo por archivo cargado.
+        upload_type_instructions: Para crear un solo trabajo para todos los archivos, vaya a
+        label: Añadir nuevo trabajo
```
Comment (jcoyne) [inline config/locales/hyrax.es.yml]: I'd say 'nueva obra' over 'nuevo trabajo'

**Comment by carrickr** [comment]
Changes made and commits squashed

**Comment by carrickr** [comment]
Done, also capitalized New File in es to be consistent with the English wording.

**Comment by jcoyne** [comment]
@carrickr 👍  now you just need to squash.

**Comment by carrickr** [comment]
Squashed, just out of curiosity there is a reason you don't use Github's automatic merge and squash feature?  I'm asking since we were going to start making it our default merge on Avalon unless there are known issues.

**Comment by jcoyne** [comment]
@carrickr I don't belive that every PR should be a single commit.  Sometimes it's valid to have multiple commits, e.g. ("Fixed bug X", "Changed to use new rubocop format").  Both commits are necessary for the build to pass, but they are both units of work we want separate.

**Pull request merged**